pandas/_testing/__init__.py

```diff
@@ -259,7 +259,7 @@ def box_expected(expected, box_cls, transpose=True):
         expected = DatetimeArray(expected)
     elif box_cls is TimedeltaArray:
         expected = TimedeltaArray(expected)
-    elif box_cls is np.ndarray:
+    elif box_cls is np.ndarray or box_cls is np.array:
```

Comment:
umm this is very strange

Reply:
we're a bit inconsistent about when we use the class vs the constructor; np.ndarray(foo) doesn't work like np.array(foo)

Reply:
ok sure, but should ideally fix this at some point.